python/tests/transformers/tf_tensor_test.py

```diff
@@ -0,0 +1,157 @@
+# Copyright 2017 Databricks, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from keras.layers import Conv1D, Dense, Flatten, MaxPool1D
+import numpy as np
+import tensorflow as tf
+import tensorframes as tfs
+
+from pyspark.sql.types import Row
+
+from sparkdl.graph.builder import IsolatedSession
+import sparkdl.graph.utils as tfx
+from sparkdl.transformers.tf_tensor import TFOneDimTensorTransformer
+
+from ..tests import SparkDLTestCase
+
+def grab_df_arr(df, output_col):
+    """ Stack the numpy array from a DataFrame column """
+    return np.array([row.asDict()[output_col]
+                     for row in df.select(output_col).toLocalIterator()])    
+
+class TFOneDimTransformerTest(SparkDLTestCase):
+
+    def _get_rand_vec_df(self, num_rows, vec_size):
+        return self.session.createDataFrame(
+            Row(idx=idx, vec=np.random.randn(vec_size).tolist())
+            for idx in range(num_rows))
+
+    def test_simple(self):
+        # Build a simple input DataFrame
+        vec_size = 17
+        num_vecs = 31
+        df = self._get_rand_vec_df(num_vecs, vec_size)
+        analyzed_df = tfs.analyze(df)
+
+        # Build the TensorFlow graph
+        with tf.Session() as sess:
+            #x = tf.placeholder(tf.float64, shape=[None, vec_size])
+            x = tfs.block(analyzed_df, 'vec')
+            z = tf.reduce_mean(x, axis=1)
+            
+            # Get the reference data
+            _results = []
+            for row in df.rdd.toLocalIterator():
+                arr = np.array(row.vec)[np.newaxis, :]
+                _results.append(sess.run(z, {x: arr}))
+            out_ref = np.hstack(_results)
+
+            # Apply the transform
+            transfomer = TFOneDimTensorTransformer(
+                tfGraph=sess.graph, inputTensor=x, outputTensor=z,
+                inputCol='vec', outputCol='outCol')
+            final_df = transfomer.transform(analyzed_df)
+            out_tgt = grab_df_arr(final_df, 'outCol')
+            
+        self.assertTrue(np.allclose(out_ref, out_tgt))
+            
+
+    def test_simple_graph_function(self):
```

Comment:
what does this test? it seems like the difference between this and test_simple() is that this one uses GraphFunction and IsolatedSession? I don't think that test belongs in this class (and the functionality should have been tested in the Suites for GraphFunction/IsolatedSession anyway).